File: TheProgrammeOfArt/DemoOfChapter7/app/src/main/java/com/example/chapter7/MainActivity.java
package com.example.chapter7;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.drawable.AnimationDrawable;
import android.os.Bundle;
import android.view.View;
import android.view.animation.AlphaAnimation;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView textView;
    
    ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = findViewById(R.id.text);
        imageView = findViewById(R.id.imageView);

        textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
//                setViewAnimationTest();
//                setAlphaAnimationTest();

                setFrameAnimationTest();
            }
        });


    }
    
    private void setFrameAnimationTest(){
        imageView.setBackgroundResource(R.drawable.frame_animation);
        AnimationDrawable animationDrawable = (AnimationDrawable) imageView.getBackground();
        animationDrawable.start();

    }

    private void setAlphaAnimationTest(){
        AlphaAnimation alphaAnimation = new AlphaAnimation(0,1);
        alphaAnimation.setDuration(3000);
        textView.startAnimation(alphaAnimation);
    }

    private void setViewAnimationTest() {
        Animation animation = AnimationUtils.loadAnimation(this,R.anim.view_animation_test);
        textView.startAnimation(animation);
    }
}

File: LittleDemo/NetworkRequestDemo/okhttpdemo/src/main/java/com/example/okhttpdemo/MainActivity.kt
package com.example.okhttpdemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.okhttpdemo.entity.BrandBean
import com.example.okhttpdemo.utils.XMLUtils
import com.thoughtworks.xstream.XStream


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        val okHttpUtils = OkHttpUtils()
//        okHttpUtils.get()
//        okHttpUtils.post()

        XMLUtils().XmlToBean1(this);
//        XMLUtils().BeanToXml();

    }




}